src/CISDSCResourceGeneration/CISDSCResourceGeneration.psm1

```diff
@@ -6,21 +6,48 @@ Get-ChildItem -Path $FunctionPath -Filter "*.ps1" -Recurse | ForEach-Object -Pro
     . $_.FullName | Out-Null
 }
 
+#This class is utilized to help centralize the logic for generating parameters for recommendations where appropriate.
+#These are created within the ScaffoldingBlock class and later used within Get-ConfigurationHeader
+#The constructor should probably be reworked to just take a scaffolding block object vs the various properties individually.
```

Comment:
Should we open a new issue from this comment?

Reply:
LOL I forgot to remove that comment. You actually already created an issue from that comment that is one of the things this addresses

Reply:
https://github.com/techservicesillinois/SecOps-Powershell-CISDSC/issues/30

Reply:
I'll make sure and remove the comment in the house keeping follow up PR to this one.